models/RwdImage.php

```diff
@@ -290,6 +323,10 @@ public function get_set_sources() {
 			return null;
 		}
 
+		if ( $this->is_svg_image() ) {
+			return;
+		}
```

Comment:
you should just allow to print this image in configured image sizes, you should not terminate method execution